vmdb/lib/miq_automation_engine/service_models/mixins/miq_ae_service_miq_request_mixin.rb

```diff
@@ -13,6 +13,7 @@ def get_option_last(key)
   end
 
   def user_message=(msg)
+    @object.update_attributes(:message => msg)
```

Comment:
@tinaafitz Should this logic be moved into the app/model/mixins/miq_request_mixin.rb method?  Also, users can pass nil to clear out the user_message so this would end up leaving the message field blank which I do not think is the expected behavior.